.github/workflows/continuous-integration.yml

```diff
@@ -88,6 +88,26 @@ jobs:
           fi
           echo $HDF5_DIR
 
+      - name: Download a specific release of PETSc
+        run: |
+          PETSC_VERSION="v3.20.5"
+          wget https://gitlab.com/petsc/petsc/-/archive/$PETSC_VERSION/petsc-$PETSC_VERSION.zip
+          unzip petsc-$PETSC_VERSION.zip
+          rm petsc-$PETSC_VERSION.zip
+          mv petsc-$PETSC_VERSION petsc
+
+      - name: Install PETSc with complex support, and test it
+        working-directory: ./petsc
+        run: |
+          ./configure --with-scalar-type=complex --with-fortran-bindings=0
```

Comment:
Just as a remark, external packages like MUMPS use Fortran (https://petsc.org/release/install/install_tutorial/), which requires an installation without the flag `--with-fortran-bindings=0`.